specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -30,6 +30,20 @@
                 "revisionName": "testcontainerApp0-ab1234",
                 "label": "production"
               }
+            ],
+            "ipSecurityRestrictionRules": [
+              {
+                "name": "name",
+                "description": "desc",
+                "ipAddress": "192.168.1.1/32",
+                "action": "Allow"
+              },
+              {
+                "name": "name",
+                "description": "desc",
+                "ipAddress": "192.168.1.1/16",
+                "action": "Deny"
```

Comment:
I believe deny is not possible in the same rule set

Reply:
fixed